echidna/output/plot.py

```diff
@@ -2,17 +2,25 @@
 import matplotlib.pyplot as plt
 import numpy
 
+import echidna.core.spectra as spectra
```

Comment:
Why is spectra imported here? It is not actually used as far as I can see. Also:

    from mpl_toolkits.mplot3d import Axes3D

can now be removed as this seems to be unused now.